deploy/csv2db.py

```diff
@@ -0,0 +1,23 @@
+import json
+import pandas as pd
+import subprocess
+from urllib.parse import urlparse
+
+from sqlalchemy import create_engine
+
+CSVPATH = 'tests/data.csv'
+
+command = 'heroku config:get DATABASE_URL'
+heroku_pg = subprocess.check_output(command.split()).decode('utf-8')
+url = urlparse(heroku_pg)
+config = {'host': url.hostname,
+          'database': url.path[1:-1],
+          'user': url.username,
+          'port': url.port, 
+          'password': url.password}
+
+engine = create_engine('postgres://', connect_args=config)
+
+df = pd.read_csv(CSVPATH)
```

Comment:
It's probably easier to use `copy_to` instead of working your way through Pandas: http://stackoverflow.com/a/22074610/176071